src/chatdlg.cpp

```diff
@@ -62,7 +62,7 @@ CChatDlg::CChatDlg ( QWidget* parent ) : CBaseDlg ( parent, Qt::Window ) // use
     pMenu->addMenu ( pEditMenu );
 #if defined( Q_OS_IOS )
     QAction* action = pMenu->addAction ( tr ( "&Close" ) );
-    connect ( action, SIGNAL ( triggered() ), this, SLOT ( close() ) );
```

Comment:
Whilst we're doing this, I would like the two lines changed to match the `addAction`s before and after that both include the action handler.

I'll also ask _why_ this should be `hide()` for iOS and `close()` for Android -- I'm going to hazard a guess it should be the same request to Qt on both platforms and that Qt should do the right thing.

Reply:
As far as I understand hide() is close() minus deleting the dialog. 
Somehow on iOS under Qt6 the dialog already gets deleted and close() may attempt to do it twice.

Potentially it's the same on android with Qt6 - but we don't know for sure yet. 

Yes, we could add the close menu entry on all OS - on desktop it seems redundant however.

Reply:
I'd argue that UI consistency across platforms isn't "redundant" but makes for a more consistent user experience.

Reply:
The close entry on desktop OS is redundant since there's a close button in the window manager. 

It does not make sense to hide the close button in a menu on mobile OS (at least iOS, where there's no back button).